### ISSUE TITLE
Create new widget (Google review)

### ISSUE THREAD

**Issue by promocot** [elementor/elementor#14651]
Hello, please add a new widget for marketing which would dynamically transmit reviews from Google to the page through the API, since many users trust Google and not simple reviews on the site, this can increase the conversion rate

**Comment by paauserrat** [comment]
I really look forward to have this feature implemented! Also, it would be nice to display this rating in the google search results next to the link or title of the page.

**Comment by paauserrat** [comment]
As there is not a Google review widget in elementor, I'm using elementor elements to recreate it. But to make it clickable I need to add custom html to the widgets to link them to Google Review.
![image](https://user-images.githubusercontent.com/73240206/122729834-e0d71580-d279-11eb-9a01-1061dd6fb378.png)
(https://github.com/elementor/elementor/issues/15423)

**Comment by promocot** [comment]
> As there is not a Google review widget in elementor, I'm using elementor elements to recreate it. But to make it clickable I need to add custom html to the widgets to link them to Google Review.
> ![image](https://user-images.githubusercontent.com/73240206/122729834-e0d71580-d279-11eb-9a01-1061dd6fb378.png)
> (#15423)

yes, today I use a plugin for google reviews and must add CSS for better design, my example
![google review](https://user-images.githubusercontent.com/66754335/122746646-e42acb00-d292-11eb-8efa-6c8f6a4f0841.png)

**Comment by promocot** [comment]
plugin - Widgets for Google Reviews, use the link to place, without API is a very simple solution, if widget from elementor will be similar I  think everybody will be happy

**Comment by SolunaCG** [comment]
+1 this would be great to have!
